lib/app/endpoint/home.rb

```diff
@@ -11,7 +11,7 @@ class Home < Endpoint
 
         # Return the index page of the application
         get '/?' do
-          erb :index, {}, modules: settings.modules.map { |m| Inferno::Module.get(m) }.reject(&:nil?)
+          erb :index, {}, modules: settings.modules.map { |m| Inferno::Module.get(m) }.reject(&:nil?), presets: settings.presets
```

Comment:
You can use `.compact` rather than `.reject(&:nil)`

Reply:
Sounds good, the `.reject(&:nil)` was from the merge conflict so I thought that was something @Jammjammjamm changed, but I will change it to `.compact`.

Reply:
Yeah, rubocop knows to change instances of `.select { !something }` to `.reject { something }`, but it doesn't know to change `.reject(&:nil?)` to `.compact`.